examples/step-85/doc/intro.dox

```diff
@@ -0,0 +1,246 @@
+@f{equation*}
+  \mathcal{T}_\Omega^h = \{ T \in \mathcal{T}^{h} : T \cap \Omega \neq \emptyset \}.
+@f}
+This is usually referred to as the "active mesh".
+@image html step-85-active-mesh.svg
+The finite element space where we want to find our numerical solution, $u_h$, is now
+@f{equation*}
+  V_\Omega^h = \{ v \in C(\mathcal{N}_\Omega^h) : v \in Q_p(T), \, T \in \mathcal{T}_\Omega^h \},
+@f}
+where
+@f{equation*}
+  \mathcal{N}_\Omega^h = \bigcup_{T \in \mathcal{T}_\Omega^h} \overline{T},
+@f}
+and $\overline{T}$ denotes the closure of $T$.
+The set $\mathcal{N}_\Omega^h$ is sometimes referred to as the "fictitious domain".
+Since $\Omega \subset \mathcal{N}_\Omega^h$,
+we see that the numerical solution is defined over a slightly larger region than the analytical solution.
+
+In this type of immersed finite element method,
+the standard way to apply boundary conditions is using Nitsche's method.
+Multiplying the PDE with a test function, $v_h \in V_h$,
+and integrating by parts over $\Omega$, as usual, gives us
+@f{equation*}
+  (\nabla u_h, \nabla v_h)_\Omega - (\partial_n u_h, v_h)_\Gamma = (f,v)_\Omega.
+@f}
+Let $\gamma_D > 0$ be a scalar penalty parameter and let $h$ be some measure of the local cell size.
+We now note that the following terms are consistent with the Dirichlet boundary condition:
+@f{align*}
+  -(u_h, \partial_n v_h)_\Gamma &= -(u_D, \partial_n v_h)_\Gamma, \\
+  \left (\frac{\gamma_D}{h} u_h, v_h \right )_\Gamma &= \left (\frac{\gamma_D}{h}u_D, v_h \right )_\Gamma.
+@f}
+Thus, we can add these to the weak formulation to enforce the boundary condition.
+This leads to the following weak formulation:
+Find $u_h \in V_h$ such that
+@f{equation*}
+  a_h(u_h, v_h) = L_h(v_h), \quad \forall v_h \in V_\Omega^h,
+@f}
+where
+@f{align*}
+  a_h(u_h, v_h) &=  (\nabla u_h, \nabla v_h)_\Omega
+                  - (\partial_n u_h, v_h)_\Gamma
+                  - (u_h, \partial_n v_h)_\Gamma
+                  + \left (\frac{\gamma_D}{h} u_h, v_h \right )_\Gamma,
+  \\
+  L_h(v_h)      &=  (f,v)_\Omega
+                  + \left (u_D, \frac{\gamma_D}{h} v_h -\partial_n v_h \right )_\Gamma.
+@f}
+In this formulation, there is one big difference,
+compared to a standard boundary-fitted finite element method.
+On each cell,
+we need to integrate over the part of the domain and the part of the boundary that falls within the cell.
+Thus, on each cell intersected by $\Gamma$,
+we need special quadrature rules that only integrate over these parts of the cell,
+that is, over $T \cap \Omega$ and $T \cap \Gamma$.
+@image html immersed_quadratures.svg
+Since $\Omega \cap T$ is the part of the cell that lies inside the domain,
+we shall refer to the following regions
+@f{align*}
+  \{x \in T : \psi(x) < 0 \}, \\
+  \{x \in T : \psi(x) > 0 \}, \\
+  \{x \in T : \psi(x) = 0 \},
+@f}
+as the "inside", "outside" and the "surface region" of the cell $T$.
+
+The above method is sometimes referred to as the "naive weak formulation"
```

Comment:
"above method" is not clear. You were introducing the notion of `inside`, `outside`, ... but you referencing what was before.